controller.go

```diff
@@ -258,7 +258,7 @@ func (c *Controller) Abort(code string) {
 
 // CustomAbort stops controller handler and show the error data, it's similar Aborts, but support status code and body.
 func (c *Controller) CustomAbort(status int, body string) {
-	c.Ctx.ResponseWriter.WriteHeader(status)
+	c.Ctx.Output.Status = status
```

Comment:
Hi, @JessonChan 
I notice that you use `Output.Status` instead of `WriteHeader`, I don't know why.
And here cause `controller.CustomAbort(status, body)` failed! It always respond status code `200` instead of `status`.